packages/buttons/stories/6-IconButton.stories.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import ChevronDownIcon from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
+
+export default {
+  title: 'Components/Buttons/IconButton',
+  component: IconButton
+} as Meta;
+
+export const Default: Story = ({
+  isDanger,
+  size,
+  isPrimary,
+  isBasic,
+  isPill,
+  focusInset,
+  isRotated
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <IconButton
+          title="Settings"
+          aria-label="Settings"
+          isDanger={isDanger}
+          size={size}
+          isPrimary={isPrimary}
+          isBasic={isBasic}
+          isPill={isPill}
+          focusInset={focusInset}
+          isRotated={isRotated}
+        >
+          <ChevronDownIcon />
+        </IconButton>
+      </Col>
+    </Row>
+  </Grid>
+);
+
+Default.argTypes = {
```

Comment:
`focusInset` is showing as a dynamic text input for this example.

Reply:
It looks like the control definition was missed. I've added it and should work as expected now 👍.